ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -409,6 +412,7 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     ///   - title: The title string
     ///   - subtitle: The subtitle string
     ///   - footer: The footer string
+    ///   - cellLeadingView: The view on the leading edge of the `customView`
```

Comment:
something about this name is repetitive. you are already in cell class, so calling cell again seem redundant. you don't say "cellTitle" for example. customLeadingView maybe?

Reply:
That was my original naming actually, but changed it because it felt also redundant with the existing customView. But maybe with making the view only a pre-set dot, I can just go ahead and name it as leadingDotView? (or the like)

Reply:
Resolved with prior convo